.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+jobs:
+  verify-release:
+    name: Verify if tag is valid
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout arsenal-networking repository
+        uses: 'actions/checkout@v3'
+        with:
+          ref: ${{ github.ref }}
+      
+      - name: Fetch tags
+        run: git fetch --tags
+      
+      - name: 'Check if tag was already created'
+        shell: bash
+        run: >
+          git show-ref --tags v${{ github.event.inputs.tag }} --quiet \
+          && exit 1 || exit 0
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs:
+    - verify-release
+    steps:
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release ${{ github.event.inputs.tag }}
+          tag_name: v${{ github.event.inputs.tag }}
```

Comment:
I think it's maybe better to stick with how we name tags in other repos to keep things more consistent (i.e without the `v`) ?

Reply:
Good catch and I agree. Updated in: https://github.com/scality/arsenal-networking/pull/3/commits/09083c281df324da03d43ffbe46fb178cbda4feb